doc/modules/preprocessing.rst

```diff
@@ -248,6 +248,43 @@ a :class:`KernelCenterer` can transform the kernel matrix
 so that it contains inner products in the feature space
 defined by :math:`phi` followed by removal of the mean in that space.
 
+.. _preprocessing_transformer:
+
+Non-linear transformation
+=========================
+
+In the contrary of scaling, data can be non-linearly transformed to
```

Comment:
"In the contrary of scaling" -> "Contrary to scaling".

However, I don't think `QuantileTransformer` should be exclusively describe in terms of outliers. It more generally may help when the feature does not have a simple parametric distribution, and where one wants to use a learner that relies on linear weighting or known metric spaces.

What also deserves emphasis here is what *is* similar between `QuantileTransformer` and scalers. Perhaps something like (please feel free to change):
```
Like scalers, :class:`QuantileTransformer` puts each feature into the same
range or distribution. However, by performing a rank transformation, it smooths
out unusual distributions and is less influenced by outliers than scaling
methods. It does, however, distort correlations and distances within and across
features.
```